.github/workflows/docker-build-and-push.yml

```
@@ -0,0 +1,415 @@
+
+      - name: Set repository name
+        id: repo
+        run: |
+          if [[ -z "${{ inputs.ecr_repository }}" ]]; then
+            # Extract repository name from github.repository
+            REPO_NAME=$(echo "${{ github.repository }}" | cut -d '/' -f 2)
+            echo "name=$REPO_NAME" >> $GITHUB_OUTPUT
+          else
+            echo "name=${{ inputs.ecr_repository }}" >> $GITHUB_OUTPUT
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)